oklayoutinflator/src/main/java/tech/okcredit/layout_inflator/OkLayoutInflater.kt

```diff
@@ -35,7 +88,8 @@ class OkLayoutInflater(context: Context) {
         }
     }
 
-    fun cancel() {
+    fun cancelInflation() {
```

Comment:
Can we keep name cancel() only? 

It's a breakout change for people who are already using the current version

Reply:
Can do but I think `cancelnflation()` looks more correct as per the operation the function is going to perform & also it does not seem to be a breaking change to me.

Let me know what you think.

Reply:
We are calling okLayoutInflator.cancel(). 
The purpose of this class is to do view inflation. It's obvious from the class's purpose that the name `cancel` makes sense to me. This same principle applies to `scope.cancel()`

This is a breakout change, and users will be unable to build their projects once they update the version due to the change.

Reply:
Makes sense.
Refactored in the latest commit.

Reply:
I actually like the cancelInflation as a method name. Breaking changes are not a problem if changelog and migration instructions cover the breaking change. Additionally this is just a nomenclature change and not a SLA/Behaviour change.

Reply:
Agreed that it is not a major breaking change, however, @anjalsaneen's point is actually correct in-terms of the functionality + function naming.

The `coroutineContext` (`SupervisorJob`) also follows the same naming convention & that's what we did.